pkg/ocicni/ocicni.go

```diff
@@ -105,7 +105,7 @@ func (plugin *cniNetworkPlugin) podUnlock(podNetwork PodNetwork) {
 	fullPodName := buildFullPodName(podNetwork)
 	lock, ok := plugin.pods[fullPodName]
 	if !ok {
-		logrus.Warningf("Unbalanced pod lock unref for %s", fullPodName)
+		logrus.Warningf("Cannot find reference in refcount map for %s. Refcount cannot be determined.", fullPodName)
```

Comment:
This looks like something fatal. Wouldn't be better to have it as Errorf if that's really the case?

Reply:
fixed!